agents/sbd/fence_sbd.py

```diff
@@ -385,7 +414,7 @@ def main():
     # If not specified then read SBD_DEVICE from environment
     if "--devices" not in options:
         dev_list = os.getenv("SBD_DEVICE")
-        if dev_list:
+        if sbd_daemon_is_running() and dev_list:
```

Comment:
If nothing appropriate is coming from SBD_DEVICE we don't have to check for the process.
Thus I'd turn the check around.